### ISSUE TITLE
Not available on Play Store

### ISSUE THREAD

**Issue by hong-kong-er** [mmazzarolo/ordinary-puzzles-app#51]
Error message when trying to download on play store - could this be fixed please?
"This app isn't available for your device because it was made for an older version of Android"

Potential solution:
https://stackoverflow.com/questions/75515890/this-app-isnt-available-for-your-device-because-it-was-made-for-an-older-versio

**Comment by mmazzarolo** [comment]
Yeah unfortunately Google made a change this year disabling access to apps targeting old SDKs. I did try updating Ordinary Puzzles early this year but it was a very long process, and I'm not sure when I'll have time to complete it :(

**Comment by hong-kong-er** [comment]
That's a shame - is there a way to install without the Play Store?

**Comment by mmazzarolo** [comment]
I think you can run it as a PWA? https://ordinarypuzzles.com/play/

**Comment by hong-kong-er** [comment]
Gotcha thanks!
